airbyte-integrations/connectors/source-github-singer/build.gradle

```diff
@@ -4,9 +4,17 @@ plugins {
 
 apply from: rootProject.file('tools/gradle/commons/integrations/image.gradle')
 apply from: rootProject.file('tools/gradle/commons/integrations/integration-test.gradle')
+apply from: rootProject.file('tools/gradle/commons/integrations/standard-test.gradle')
 
 dependencies {
 }
 
+standardTest{
+    ext {
```

Comment:
can we also add the ability to configure a before/after each/all script?

Reply:
Although I'm realizing this might be a hitch because we might need to pass new resources in these scripts e.g: if I standup a new DB in the beforeEach, I'd need to pass the credentials every time. Would it make sense to rewrite the config file? Locks us into single threaded test runs but seems like a problem for later.

Reply:
> can we also add the ability to configure a before/after each/all script?

yes. see PR message. that's the next task to do. going to do it in a separate pr.